lib/platform/bitbucket-server/index.md

```diff
@@ -1,9 +1,10 @@
 # Bitbucket Server Support
 
+> Note: For Assignees support use username slug as asignee name. Which can be easy found here: `https://{bitbucket_server}/rest/api/1.0/users`
```

Comment:
You should note the required server version on which that was made available.

Please write one sentence per line.

Reply:
I don't think this point is worth of putting directly under the heading. Please find a more normal location for it, such as at the bottom

Reply:
Good points, I moved it and added supported versions (every bitbucket version)